components/pipeline-service/production/base/main-pipeline-service-configuration.yaml

```diff
@@ -601,7 +608,7 @@ data:
     LOGS_API=false
     LOGS_TYPE=File
     LOGS_BUFFER_SIZE=5242880
-    LOGS_PATH=//logs
+    LOGS_PATH=/logs
```

Comment:
Why was that value changed? I see further below we also downgrade (reference older image) of Results, why?